### PR TITLE
[Docs][UI Next] Add package license.

### PR DESCRIPTION
**Pull request by songjianet**
## Purpose of the pull request
[#7332](https://github.com/apache/dolphinscheduler/issues/7332)

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by zhongjiajie** [comment]
Should  we also add to `License` file? @CalvinKirs

**Review by CalvinKirs — DISMISSED** [review]
you also need license to `dolphinscheduler-dist/release-docs/LICENSE`
and add NOTICE to `dolphinscheduler-dist/release-docs/NOTICE`
In addition, please make sure that the corresponding license corresponds to the version you are referencing

**Comment by songjianet** [comment]
> you also need license to `dolphinscheduler-dist/release-docs/LICENSE` and add NOTICE to `dolphinscheduler-dist/release-docs/NOTICE` In addition, please make sure that the corresponding license corresponds to the version you are referencing

I know I have to deal with this, but I'm working on other things now, I can merge these first, and I'll add them later.

**Review by zhongjiajie — APPROVED** [review]
lgtm

File: dolphinscheduler-dist/release-docs/licenses/ui-next-licenses/LICENSE-eslint [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright OpenJS Foundation and other contributors, <www.openjsf.org>
```
Comment (kezhenxu94) [inline dolphinscheduler-dist/release-docs/licenses/ui-next-licenses/LICENSE-eslint:1]: Dependencies in `devDependencies` don't need to be included in the release licenses

Comment (songjianet) [reply dolphinscheduler-dist/release-docs/licenses/ui-next-licenses/LICENSE-eslint:1]: This piece may be adjusted in the future, so I wrote all the dependencies.

**Pull request merged**